manifests/Hyperhue-Studios.yaml

```diff
@@ -0,0 +1,13 @@
+name: HyperHue-Studios
+authors: ChemtrailOperator
+homepage: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/tree/main
+license: GPL-3.0-or-later
+version: v1.4
+shortDescription: In-Game access to all 29 vanilla swizzles
+description: This plugin creates game play to swizzle your ship to any of the 29 vanilla swizzles. To begin, visit any human planet.
+url: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/releases/download/v1.4/HyperHue-Studios-1.4.zip
+iconUrl: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/blob/main/v1.4/icon.png
+autoupdate:
+  type: tag
+  url: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/releases/download/$version/HyperHue-Studios-1.4.zip
```

Comment:
This will be a problem, since you have "1.4" as part of your filename. Two options:

1. Remove the version from future filenames (`HyperHue-Studios.zip`)
2. Use the version in the filename, but make sure it's the exact same as the tag name (`v1.4`, **not** `1.4`). Then the filename could be `HyperHue-Studios-v1.4.zip`, and the autoupdate url `...download/$version/HyperHue-Studios-$version.zip`